build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -933,6 +933,13 @@ message SymlinkNode {
 message Digest {
   // The hash. In the case of SHA-256, it will always be a lowercase hex string
   // exactly 64 characters long.
+  //
+  // For any digest function having enumeration value 8 or higher, the hash MUST
+  // be prefixed with the shortest decimal representation of the enumeration
```

Comment:
If we go with this encoding, should we also specify that the hash must not contain prefixes for values 7 or under?

Reply:
I think that the sentence at the start of the comment sort of implies this:

> In the case of SHA-256, it will always be a lowercase hex string exactly 64 characters long.

Though that only covers the SHA-256 case. Maybe the right thing is to specify it even further, stating that all hashes must be a lowercase hex string. But then again, I don't know what the convention is for VSO, for example.